homeassistant/components/binary_sensor/mychevy.py

```diff
@@ -0,0 +1,68 @@
+"""Support for MyChevy sensors."""
+
+import asyncio
+from logging import getLogger
+
+from homeassistant.components.mychevy import (
+    EVBinarySensorConfig, DOMAIN,
+)
+from homeassistant.components.binary_sensor import (
+    ENTITY_ID_FORMAT, BinarySensorDevice)
+from homeassistant.util import (slugify)
+
+_LOGGER = getLogger(__name__)
+
+SENSORS = [
+    EVBinarySensorConfig("Plugged In", "plugged_in", "plug")
```

Comment:
The ususal way to do this in home assistant is to use a dict holding a list of name, unit, icon etc. See eg:
https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/sensor/luftdaten.py#L38-L44

I don't think you need to create a class for the config. I like the verbosity of attribute access that a class gives but I think we should try to stick within the mold as long as the mold will fit. If you want to keep the verbose attribute access you can create a nested list of dicts, instead of a nested list of lists.

```py
SENSORS = [
    {'name': 'EVCharge', 'attr': 'percent', 'unit_of_measurement': '%', 'icon': 'mdi:battery'}
   ...
]
```